css/menu.css

```diff
@@ -2,7 +2,17 @@
 .fl-with-bottom-menu {
   -webkit-transition: padding 0.2s;
   transition: padding 0.2s;
-  padding-bottom: calc(44px + constant(safe-area-inset-bottom));
+  padding-bottom: 44px;
+  padding-bottom: calc(44px + env(safe-area-inset-bottom));
+}
```

Comment:
In the case of `.supports-container`, both `.fl-with-bottom-menu` and `.fl-page-content-wrapper` still end up with paddings at the bottom. I thought the idea was to  make sure they don't both end up with these paddings?

Reply:
Removed duplicate css.